src/main/kotlin/example/ColorMode.kt

```diff
@@ -0,0 +1,45 @@
+package example
+
+import kotlinext.js.jso
+import mui.material.styles.*
+import react.*
+
+enum class PaletteMode(val paletteOptions: PaletteOptions) {
```

Comment:
Check file `Sizes.kt` and reorganize it in the same way.
Seems it can be called `Themes`